plugin/plugins/peerlog/peerlog.go

```diff
@@ -60,6 +63,38 @@ func (*peerLogPlugin) Start(node *core.IpfsNode) error {
 		)
 	}
 	node.PeerHost.Network().Notify(&notifee)
+
+	sub, err := node.PeerHost.EventBus().Subscribe(
+		new(event.EvtPeerIdentificationCompleted),
+		eventbus.BufSize(1024),
```

Comment:
for learnings, is there any reason for 1024 here, or just a nice number?

Reply:
It's a nice round number. Basically, if we slow down handling these events and fill this buffer, identifying new peers may block. This gives us some wiggle room in case logging slows down.